docs/devel/kubectl-conventions.md

```diff
@@ -110,7 +110,7 @@ Updated: 8/27/2015
   * However, affordances are made for simple parsing of `get` output
 * Only errors should be directed to stderr
 * `get` commands should output one row per resource, and one resource per row
-  * Column titles and values should not contain spaces in order to facilitate commands that break lines into fields: cut, awk, etc.
+  * Column titles and values should not contain spaces in order to facilitate commands that break lines into fields: cut, awk, etc. Instead, use `_` as the word separator.
```

Comment:
You use dash. :-)

Could you change this to dash and then also change CLUSTER_IP and EXTERNAL_IP to match the convention?